packages/scene-composer/src/components/panels/AddComponentMenu.tsx

```diff
@@ -134,6 +157,15 @@ export const AddComponentMenu: React.FC<AddComponentMenuProps> = ({ onSelect })
     addComponentInternal(selectedSceneNodeRef, component);
   }, [selectedSceneNodeRef, selectedSceneNode]);
 
+  const handleAddAnimations = useCallback(() => {
+    if (!selectedSceneNodeRef) return;
+
+    let selector = animationComponent.selector || 0;
```

Comment:
Give a proper variable name? 
```animationSelector``` and why are you incrementing it by 1? Is it always incrementing by 1?